lib/tasks/index.rake

```diff
@@ -0,0 +1,54 @@
+require 'lockfile'
+
+INDEX_DIR = Pathname.new('/web-archiving-stacks/data/indices/cdx').freeze
+
+namespace :index do
```

Comment:
i'm wondering about cdx_index instead?  Or wayback_index?  or crawl_index?   index is so ... overloaded